.bazelversion

```diff
@@ -1 +1 @@
-7.4.1
+8.0.1
```

Comment:
8.1.0 is out now, so let's use that instead.

Reply:
Done! Now, we're using our custom version of Bazel, which is based on v8.1.1.